public/achievementInfo.php

```diff
@@ -189,9 +189,14 @@ function ResetProgress() {
         echo "<div id='achievemententry'>";
 
         $renderedTitle = renderAchievementTitle($achievementTitle);
+
         echo "<div class='flex justify-between'>";
         echo "<div>";
-        echo "<a href='/achievement/$achievementID'><strong>$renderedTitle</strong></a> ($achPoints)<span class='TrueRatio'> ($achTruePoints)</span><br>";
+        echo "<a href='/achievement/$achievementID'><strong>$renderedTitle</strong></a>";
+        if ($achPoints !== 0 && $achTruePoints !== 0) {
```

Comment:
This will also hide points on items that have points but true points haven't been calculated yet.